plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -975,6 +989,57 @@ public static void verifyExtraProperties(Set<String> basePropertyKeys, Map<Strin
         }
     }
 
+    public static OptionalInt getCompressionLevel(IcebergFileFormat fileFormat, Map<String, String> storageProperties)
+    {
+        if (!VALID_ICEBERG_FILE_FORMATS_FOR_COMPRESSION_LEVEL_PROPERTY.contains(fileFormat)) {
+            return OptionalInt.empty();
+        }
+        String compressionLevelProperty = getCompressionLevelName(fileFormat);
+        String compressionLevelValue = storageProperties.get(compressionLevelProperty);
+        if (!Strings.isNullOrEmpty(compressionLevelValue)) {
```

Comment:
I would just inline `compressionLevelValue != null && !compressionLevelValue.isEmpty()` instead of bringing in `Strings` to do that

Reply:
done